src/main/java/org/jfrog/hudson/pipeline/common/executors/MavenGradleEnvExtractor.java

```diff
@@ -3,95 +3,23 @@
 import hudson.EnvVars;
 import hudson.FilePath;
 import hudson.Launcher;
-import hudson.model.Result;
 import hudson.model.Run;
 import hudson.model.TaskListener;
-import org.jfrog.build.api.BuildInfoConfigProperties;
-import org.jfrog.build.api.BuildInfoFields;
 import org.jfrog.build.extractor.clientConfiguration.ArtifactoryClientConfiguration;
-import org.jfrog.hudson.CredentialsConfig;
-import org.jfrog.hudson.action.ActionableHelper;
-import org.jfrog.hudson.pipeline.common.Utils;
 import org.jfrog.hudson.pipeline.common.types.buildInfo.BuildInfo;
 import org.jfrog.hudson.pipeline.common.types.deployers.Deployer;
 import org.jfrog.hudson.pipeline.common.types.resolvers.Resolver;
-import org.jfrog.hudson.release.ReleaseAction;
-import org.jfrog.hudson.util.CredentialManager;
-import org.jfrog.hudson.util.ExtractorUtils;
-import org.jfrog.hudson.util.ResolverContext;
-import org.jfrog.hudson.util.publisher.PublisherContext;
 
 /**
- * Created by Tamirh on 04/08/2016.
+ * @author yahavi
  */
-public class MavenGradleEnvExtractor implements Executor {
-
-    private Deployer publisher;
-    private Resolver resolver;
-    private Run build;
-    private BuildInfo buildInfo;
-    private TaskListener buildListener;
-    private Launcher launcher;
-    private FilePath tempDir;
-    private EnvVars env;
+public class MavenGradleEnvExtractor extends EnvExtractor {
```

Comment:
Do we still need this class? Can we remove it and stay with *EnvExtractor* only?

Reply:
We need it for `addExtraConfiguration`